### PR TITLE
Take down defunct Writing Extensions guide

### PR DESCRIPTION
**Pull request by VSEphpbb**
This could probably wait for all the Dev Doc PRs on extension tutorials to be merged.

https://github.com/phpbb/documentation/pulls

This PR takes down the content from the now defunct Writing extensions page, and uses it as a landing page to redirect users to the NEW docs.

Also updated links and removed some never used links.

### CODE REVIEW

**Comment by marc1706** [comment]
You forgot to update the tests.

File: src/AppBundle/Resources/views/Styles/create.html.twig [hunk 1]
```diff
@@ -152,7 +152,7 @@ parent = prosilver Special Edition</pre>
 		<p>Besides these elements, phpBB 3.1.x requires styles to include certain specific pieces of code for the style to function properly.</p>
 		<p>If you are using a valid 3.1.x style as a base for your new style, these elements will be included already, but since they are very important, we will dedicate this section to a more detailed explenation.</p>
 
-		<div class="note-box blue"><strong>Note:</strong> For more information on how these code snippets are used by styles and extensions, visit the <a href="{{ extensions_writing }}">Writing Extensions</a> page or the Development Wiki page on <a href="https://wiki.phpbb.com/New_Template_Commands_in_3.1">New Template Commands in 3.1</a>.</div>
+		<div class="note-box blue"><strong>Note:</strong> For more information on how these code snippets are used by styles and extensions, visit the Development Wiki page on <a href="https://wiki.phpbb.com/New_Template_Commands_in_3.1">New Template Commands in 3.1</a>.</div>
```
Comment (paul999) [inline src/AppBundle/Resources/views/Styles/create.html.twig]: You should keep a list somewhere regarding those wiki links

**Comment by paul999** [comment]
Looks good to me :)

File: src/AppBundle/Resources/views/Styles/create.html.twig (hunk 1 above)
Comment (paul999) [inline src/AppBundle/Resources/views/Styles/create.html.twig]: Well, the only thing I get from chocolate is a allergic attack :P 

But good, was just making sure that we aren't going to link in new pages to content which gets removed next (phpBB) week.

**Comment by michaelcullum** [comment]
Ping when you want a merge

**Comment by VSEphpbb** [comment]
@michaelcullum If you set up a rewrite on our server config that will redirect `https://area51.phpbb.com/docs/dev/current/*` to `https://area51.phpbb.com/docs/dev/31x/*` we can just link to the "current" docs, and update the redirect from 31x to 32x in the future...instead of all the various links.

**Comment by VSEphpbb** [comment]
@michaelcullum this is good to merge

**Pull request merged**